sensors/Bh1750.h

```diff
@@ -13,7 +13,7 @@
 namespace as {
 
 // https://github.com/claws/BH1750
-template <byte ADDRESS=0x23,::BH1750::Mode MODE=::BH1750::Mode::CONTINUOUS_HIGH_RES_MODE>
+template <byte ADDRESS=0x23,::BH1750::Mode MODE=::BH1750::Mode::ONE_TIME_HIGH_RES_MODE>

```

Comment:
How much saves this ?

Reply:
I haven't tested how much this saves. I can try to hook up a laboratory power supply and see if it's able to measure it, or is there an easier way to do this?

Reply:
I measured it now, it takes 1.35mA at 2.4V with continious and 1.18mA at 2.4V with one time.